scripts/vagrant/zeppelin-dev/README.md

```diff
@@ -0,0 +1,154 @@
+## Vagrant Virtual Machine for Apache Zeppelin
+  
+This script creates a virtual machine that launches a repeatable, known set of core dependencies required for developing Zeppelin.  It can also be used to run an existing Zeppelin build if you don't plan to build from source.  For pyspark users, this script also includes several helpful [Python Libraries and one obscure configuration to help with matplotlib plotting inside Zeppelin](#pythonextras)
+ 
+####Installing the required components to launch a virtual machine.
+
+This script requires three applications, [Ansible](http://docs.ansible.com/ansible/intro_installation.html#latest-releases-via-pip "Ansible"), [Vagrant](http://www.vagrantup.com/downloads "Vagrant") and [Virtual Box](https://www.virtualbox.org/ "Virtual Box").  All of these applications are freely available as Open Source projects and extremely easy to set up on most operating systems.
+
+### Create a Zeppelin Ready VM in 4 Steps
+
+1. Download and Install Vagrant:  [Vagrant Downloads](http://www.vagrantup.com/downloads)
+2. Install Ansible:  [Ansible Python pip install](http://docs.ansible.com/ansible/intro_installation.html#latest-releases-via-pip)  
+`sudo easy_install pip` then   
+`sudo pip install ansible`
+3. Install Virtual Box: [Virtual Box Downloads](https://www.virtualbox.org/ "Virtual Box")
+4. Type `vagrant up`  from within the `/scripts/vagrant/zeppelin-dev` directory
```

Comment:
you might want to mention this is Mac/Linux? think we have some Windows users - eg. need to install python, change path separator etc